File: Source/TribuFu.java
// Copyright (c) TribuFu. All Rights Reserved

package tribufu;

public class TribuFu {
    /**
     * A native method that is implemented by the 'TribuFu' native library.
     */
    public native String Hello(String input);

    // Used to load the 'TribuFu' library on application startup.
    public void LoadLibrary() {
        System.loadLibrary("TribuFu");
    }
}

File: lib/src/main/kotlin/com/tribufu/sdk/TribuFu.kt
// Copyright (c) TribuFu. All Rights Reserved

package com.tribufu.sdk

class TribuFu {
    /**
     * A native method that is implemented by the 'TribuFu' native library.
     */
    external fun Hello(input: String): String

    // Used to load the 'TribuFu' library on application startup.
    init {
        System.loadLibrary("TribuFu_jvm")
    }
}
